control-plane/pkg/reconciler/channel/channel.go

```diff
@@ -352,6 +382,90 @@ func (r *Reconciler) finalizeKind(ctx context.Context, channel *messagingv1beta1
 	return nil
 }
 
+func (r *Reconciler) reconcileSubscribers(ctx context.Context, kafkaClient sarama.Client, kafkaClusterAdmin sarama.ClusterAdmin, channel *messagingv1beta1.KafkaChannel, channelContractResource *contract.Resource) error {
+	logger := kafkalogging.CreateReconcileMethodLogger(ctx, channel)
+
+	channel.Status.Subscribers = make([]v1.SubscriberStatus, 0)
+	var globalErr error
+	for _, s := range channel.Spec.Subscribers {
+		logger = logger.With(zap.Any("subscription", s))
```

Comment:
```suggestion
	for _, s := range channel.Spec.Subscribers {
		s := s // capture loop variable
		logger = logger.With(zap.Any("subscription", s))
```

Reply:
I did it differently, but thanks for pointing at the problem